doc/docker.md

```diff
@@ -0,0 +1,154 @@
+# Docker toolbox & docker-zipkin
```

Comment:
you might want to add "on Mac OS X" because parts are mac os only (eg, `brew install` stuff)